app/modules/ads/index.js

```diff
@@ -20,6 +20,12 @@ class Ads {
     this.isLoaded = false;
     this.isFastboot = typeof FastBoot !== 'undefined';
     this.onReadyCallbacks = [];
+
+    this._readyResolve = null;
```

Comment:
why underscore? that about `/** @private */`

Reply:
Good point. I can't wait for https://developer.mozilla.org/en-US/docs/Archive/Add-ons/Add-on_SDK/Guides/Contributor_s_Guide/Private_Properties